### DISCUSSION TITLE
403 Forbidden Error

### DISCUSSION THREAD

**Discussion by kwadejeffrey** [opcodesio/log-viewer#384] [Q&A]
When I navigate to the log-viewer page, I get a 403 error message.
![Screenshot 2024-07-05 230015](https://github.com/opcodesio/log-viewer/assets/102606134/a12b8363-84f8-4dc9-a99d-b9e0c39bf8d1)

**Comment by arukompas** [comment]
hey @kwadejeffrey , see my comment here:
https://github.com/opcodesio/log-viewer/discussions/337#discussioncomment-9973330

Closing this as duplicate :)
